docs/user/tutorials/01-30-10-aws-vpc-peering.md

```diff
@@ -16,19 +16,18 @@ This tutorial explains how to create a Virtual Private Cloud (VPC) peering conne
     export AWS_REGION={REGION}
     ```
 
-2. Create a trust policy document.
+2. Create a trust policy document. See [AwsVpcPeering Custom Resource](../resources/04-30-10-aws-vpc-peering.md#authorization) to identify Cloud Manager principal:
```

Comment:
I would create a separate doc on the Authorization, etc. instead of having everything in the resources document. We could use the details from the Authorization section as input. TBD in the next PR.